build/config/android/config.gni

```diff
@@ -25,22 +16,6 @@ if (is_android) {
     android_sdk_version = default_android_sdk_version
     android_sdk_build_tools_version = default_android_sdk_build_tools_version
 
-    android_default_keystore_path =
```

Comment:
nit: if you're removing these, maybe remove them from the declare_args block above too?

Reply:
Could you clarify? These deletions are the deletions from the `declare_args` block.